src/classes/Statblock.ts

```diff
@@ -100,80 +77,123 @@ class Statblock {
                 output += ` ${damages.join(' ')}`
               }
             }
-            if (i > 0 && (i + 1) % 2 === 0 && i + 1 !== loadout.Items.length) {
-              output += '\n  '
-            } else if (i + 1 < loadout.Items.length) {
-              output += ', '
-            }
           }
         }
       }
-    }
 
-    if (pilot && mech) output += '\n----------\n'
+      const bond = pilot.BondController
+      if (bond.Bond) {
+        output += '[ BOND ]\n  '
+        output += `${bond.Bond.Name.toUpperCase()}\n`
+        if (bond.BondPowers) {
+          output += '  Powers: '
+          for (let i = 0; i < bond.BondPowers.length; i++) {
+            output += `${bond.BondPowers[i].name.toUpperCase()}${linebreak(i, bond.BondPowers.length)}`
+          }
+        }
+      }
 
-    if (mech) {
-      output += `« ${mech.Name.toUpperCase()} »\n[ ${mech.Frame.Source} ${mech.Frame.Name} ]\n`
-      if (!pilot)
-        output += `H:${mech.Hull} A:${mech.Agi} S:${mech.Sys} E:${mech.Eng} SIZE:${mech.Size}\n`
-      output += `  STRUCTURE:${mech.CurrentStructure}${
-        mech.IsActive ? '/' + mech.MaxStructure : ''
-      }`
-      output += ` HP:${mech.CurrentHP}${mech.IsActive ? '/' + mech.MaxHP : ''}`
-      output += ` ARMOR:${mech.Armor}\n`
-      output += `  STRESS:${mech.CurrentStress}${mech.IsActive ? '/' + mech.MaxStress : ''}`
-      output += ` HEAT:${mech.CurrentHeat}${mech.IsActive ? '/' + mech.HeatCapacity : ''}`
-      output += ` REPAIR:${mech.CurrentRepairs}${mech.IsActive ? '/' + mech.RepairCapacity : ''}\n`
-      output += `  ATK BONUS:${mech.AttackBonus} TECH ATK:${mech.TechAttack} LTD BONUS:${mech.LimitedBonus}\n`
-      output += `  SPD:${mech.Speed} EVA:${mech.Evasion} EDEF:${mech.EDefense} SENS:${mech.SensorRange} SAVE:${mech.SaveTarget}\n`
+      if (view == "full") {output += '***\n'}
 
-      output += '[ WEAPONS ]\n'
-      for (const im of mech.FeatureController.IntegratedWeapons) {
-        output += '  INTEGRATED MOUNT: '
-        output = addWeaponToOutput(output, discordEmoji, im)
-        output += '\n'
+      if (view == "pilotBuild"){ 
+        output += '[ MECH SKILLS]\n  '
```

Comment:
Missing a space between `SKILLS` and `]` here.

Reply:
Good eye, fixed :)